nxc/protocols/ftp/proto_args.py

```diff
@@ -3,5 +3,7 @@ def proto_args(parser, std_parser, module_parser):
     ftp_parser.add_argument("--port", type=int, default=21, help="FTP port (default: 21)")
 
     cgroup = ftp_parser.add_argument_group("FTP Access", "Options for enumerating your access")
-    cgroup.add_argument("--ls", action="store_true", help="List files in the directory")
+    cgroup.add_argument("--ls", metavar="DIRECTORY", nargs="?", const=".", help="List files in the directory")
+    cgroup.add_argument("--get", metavar="FILE", help="Download a file")
+    cgroup.add_argument("--put", metavar=("LOCAL_FILE", "REMOTE_FILE"), nargs=2, help="Upload a file")
```

Comment:
Would be nice if you could add an example as in:
https://github.com/Pennyw0rth/NetExec/blob/66e6c95f0d4cabe4acabdfadaa69f8eb38bb056f/nxc/protocols/smb/proto_args.py#L75C45-L75C45

Reply:
Added: https://github.com/RomanRII/NetExec/commit/901b8dee80fc1109414c688f491f5ad4fdec9049